python/lsst/fgcmcal/fgcmOutputProducts.py

```diff
@@ -771,7 +772,7 @@ def _getConstantPhotoCalib(self, zeropoint, err, offset, scaling):
         # Take the zeropoint, apply the absolute relative calibration offset,
         # and whatever flat-field scaling was applied
 
-        calibMean = 10.**(zeropoint / (-2.5)) * 10.**(offset / (-2.5)) * scaling
+        calibMean = (offset*units.ABmag).to_value(units.nJy) * scaling
```

Comment:
See above.

Reply:
Okay, this it turns out is untested, because I only run through the "variable calibration" part of the code (because that's going to be the standard way of running it).  I will have to think of a way of testing this branch on its own without taking too long.  In any event, the line should be:
```
calibMean = ((zeropoint + offset)*units.ABmag).to_value(units.nJy) * scaling
```